pkg/broker/util.go

```diff
@@ -53,3 +53,16 @@ func SpecToService(spec *apb.Spec) Service {
 	copy(retSvc.Tags, spec.Tags)
 	return retSvc
 }
+
+func StateToLastOperation(state apb.State) LastOperationState {
+	switch state {
+	case apb.StateInProgress:
```

Comment:
Since the strings for apb.StateInProgress and LastOperationsStateInProgress are the same you could just return apb.StateInProgress.